xclim/sdba/_adjustment.py

```diff
@@ -36,7 +38,9 @@ def _adapt_freq_hist(ds, adapt_freq_thresh):
     hist_q=[Grouper.PROP, "quantiles"],
     scaling=[Grouper.PROP],
 )
-def dqm_train(ds, *, dim, kind, quantiles, adapt_freq_thresh) -> xr.Dataset:
+def dqm_train(
+    ds, *, dim, kind, quantiles, adapt_freq_thresh, jitter_under_thresh_value
+) -> xr.Dataset:
```

Comment:
Maybe there's a better argument name. What seems more natural `jitter_under_thresh` is already used for the name of the function that does the jittering, so I thought it was best not to do this. 

Now I can't remember if this change was 100% necessary, but it was at least more practical to be able to include this option in the kwargs. I think that philosophically, 'adapt_freq' and 'jitter_under' are part of the 'training' and including 'adapt_freq' was necessary as we need to compare the zeroes of the grouped `ref, hist`. Again, I'm not sure this is 100% necessary for 'jitter_under_thresh', so I can revisit this aspect if preferable